langs/te.json

```diff
@@ -1,5 +1,5 @@
 {
     "name": "Telugu",
     "code": "te",
-    "maintainers": ["ashr81", "KoushikKumar", "daminisatya", "ChaituKNag"]
+    "maintainers": ["ashr81", "KoushikKumar", "daminisatya", "ChaituKNag", "sgudik"]
```

Comment:
@sntgdk , please check this should be your GitHub username. It is `sntgdk` here it's given `sgudik `. right?